engine/samples/cars/page.md

```diff
@@ -0,0 +1,58 @@
+# Cars {#examples-engine-cars}
+
+@brief Sample that spawns multiple assets and updates them every frame.
```

Comment:
```suggestion
@brief Loading voxel grids and rendering them.
```

Perhaps? Spawn may suggest erroneously that we're using blueprints in some way or another.